src/bp/core/app/server_utils.ts

```diff
@@ -0,0 +1,44 @@
+import { Request } from 'express-serve-static-core'
```

Comment:
I'm unsure about that, I like having one level of folder there, otherwise it means moving the file server.ts to server/index.ts and i'm not sure i'd like it to be named "index" :/

Reply:
I understand your point here, but I simply wanted to illustrate the fact that underscores were only used with db table files. If we follow our "naming convention" this file should be named `server-utils.ts`, unless I'm missing something here?

Reply:
Ah good point, I missed that one